### PR TITLE
Update Design System for steps/conviction/compensation_paid_amount

### PR DESCRIPTION
**Pull request by Davidslv**
Story: https://mojdigital.teamwork.com/#/tasks/21707829

### Before
Screenshot 2020-09-02 at 08 54 49

### After

Screenshot 2020-09-02 at 08 54 31

### CODE REVIEW

File: app/views/steps/conviction/compensation_paid_amount/edit.html.erb [hunk 1]
```diff
@@ -1,23 +1,15 @@
 <% title t('.page_title') %>
+<%= step_header %>
```
Comment (zheileman) [inline app/views/steps/conviction/compensation_paid_amount/edit.html.erb:2]: Super minor thing, it doesn't really affect in any visible way but for the `step_header` we can now use the same syntax as with the `title` (removing the `=`) as it does not output anything in place, it just assigns the content for the `back_link`

File: app/views/steps/conviction/compensation_paid_amount/edit.html.erb [hunk 2]
```diff
@@ -1,23 +1,15 @@
...
 
-<div class="govuk-width-container">
-  <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
+    <%= step_subsection %>
```
Comment (zheileman) [inline app/views/steps/conviction/compensation_paid_amount/edit.html.erb:7]: I think for now it is ok to continue using this helper, until we've finished all the steps. Then we can go back and change this to use the native `caption` that the gem offers.

**Review by zheileman — APPROVED** [review]
👍 💯 
Couple of minor comments but not affecting the PR.

File: app/views/steps/conviction/compensation_paid_amount/edit.html.erb (hunk 2 above)
Comment (Davidslv) [reply app/views/steps/conviction/compensation_paid_amount/edit.html.erb]: Updated on https://github.com/ministryofjustice/disclosure-checker/pull/377/commits/3e9fc6b34d11c19de05b11512015dd310228b2d3

File: app/views/steps/conviction/compensation_paid_amount/edit.html.erb (hunk 1 above)
Comment (Davidslv) [reply app/views/steps/conviction/compensation_paid_amount/edit.html.erb]: addressed

File: app/views/steps/conviction/compensation_paid_amount/edit.html.erb (hunk 2 above)
Comment (Davidslv) [reply app/views/steps/conviction/compensation_paid_amount/edit.html.erb:7]: Got a story for this one: https://mojdigital.teamwork.com/#/tasks/21724564

**Review by njseeto — APPROVED** [review]
nice! ✨

**Pull request merged**